src/index/board_card.jsx

```diff
@@ -70,7 +70,7 @@ let BoardCard = React.createClass({
           <div className="title">
             {this.props.data.id
               .split('/')
-              .slice(2)
+              .slice(-1)
```

Comment:
Does it get too long otherwise?

Reply:
It did for that test report. 

It could also be handled by resizing the text, if you want to preserve the title with the project path.

Reply:
Makes sense, let's not go down a style tweaks rabbit hole as it seems fine like this.